src/processors/controllers/processorController.js

```diff
@@ -54,3 +54,31 @@ module.exports.redownloadJSProcessor = async function (req, res, next) {
         next(e);
     }
 };
+
+module.exports.updateProcessor = async function (req, res, next) {
+    let { body: updatedProcessor, params: { processor_id: processorId }, query: { dl = false } } = req;
+    try {
+        dl = convertDlQueryParamToBoolean(dl);
+        const processor = await processorManager.updateProcessor(processorId, updatedProcessor, dl);
+        res.status(200).json(processor);
+    } catch (e) {
+        next(e);
+    }
+};
+
+function convertDlQueryParamToBoolean(dl) {
+    const error = new Error(`dl query parameter expected one of ["true", "false"], but got ${dl}`);
```

Comment:
can remove this error (line 70 and 71). The swagger validator will throw a 400 error like this:
```
{
    "message": "Input validation error",
    "validation_errors": [
        "query/dl should be boolean"
    ]
}
```

Reply:
done